tools/cli/src/helpers/common.ts

```diff
@@ -119,7 +119,6 @@ function resolveSourceMapPath(sourceProcessor: SourceProcessor) {
         return pipe(
             asset.path,
             (path) => sourceProcessor.getSourceMapPathFromSourceFile(path),
-            R.map((result) => result ?? pathIfExists(`${asset.path}.map`)),
```

Comment:
are you sure about that?

Reply:
Yes, the `sourceProcessor` now handles that.